bower.json

```diff
@@ -14,7 +14,7 @@
     "tests"
   ],
   "dependencies": {
-    "hello": "git@github.com:dan-silver/hello.js.git#master",
```

Comment:
No changes are needed on your side for this one. Kevin already setup the site to pull hello.js from that forked repo.